.github/workflows/actions.yml

```diff
@@ -84,32 +88,32 @@ jobs:
           if [ "${IS_RC}" = "true" ]; then CONDA_LABEL="rc"; fi
           anaconda upload --label $CONDA_LABEL noarch/*.tar.bz2
 
-  trigger-deploy:
-    runs-on: ubuntu-22.04
-    needs: [linux]
-    # only trigger deploys from protected branches and tags
-    if: ${{ github.ref_protected || github.ref_type == 'tag' }}
-    steps:
-      - name: Determine Environment
-        uses: neutrons/branch-mapper@v2
-        id: conda_env_name
-        with:
-          prefix: mantidtotalscattering
-
-      - name: Trigger deploy
-        id: trigger
-        uses: eic/trigger-gitlab-ci@v2
-        with:
-          url: https://code.ornl.gov
-          token: ${{ secrets.GITLAB_TRIGGER_TOKEN }}
-          project_id: 7835
-          ref_name: main
-          variables: |
-            PLAY="update"
-            CONDA_ENV="${{ steps.conda_env_name.outputs.name }}"
-
-      - name: Annotate commit
-        uses: peter-evans/commit-comment@v2
-        with:
-          body: |
-            GitLab pipeline for ${{ steps.conda_env_name.outputs.name }} has been submitted for this commit: ${{ steps.trigger.outputs.web_url }}
+#  trigger-deploy:
+#    runs-on: ubuntu-22.04
+#    needs: [linux]
+#    # only trigger deploys from protected branches and tags
+#    if: ${{ github.ref_protected || github.ref_type == 'tag' }}
```

Comment:
Make a copy of this line that is just 
```
if: false
```
and you won't have to comment out the whole block